src/modules/SSHAuthorizedKeys.rb

```diff
@@ -39,6 +45,27 @@ def read_keys(home)
     # @see Yast::Users::SSHAuthorizedKeyring#write_keys
     def write_keys(home)
       keyring.write_keys(home)
+    rescue Users::SSHAuthorizedKeyring::HomeDoesNotExist
+      log.warn("Home directory '#{home}' does not exist")
+      Report.Warning(
+        # TRANSLATORS: '%s' is a directory path
+        _(format("Home directory '%s' does not exist\n" \
+                 "so authorized keys won't be written.", home))
```

Comment:
IIRC abbreviations are used in informal language, in computer UI we should avoid that.